examples/GetBucketVersioning.java

```diff
@@ -40,12 +41,16 @@ public static void main(String[] args)
       //         .credentials("YOUR-ACCESSKEY", "YOUR-SECRETACCESSKEY")
       //         .build();
 
-      // Disable versioning on 'my-bucketname'.
-      minioClient.disableVersioning(
-          DisableVersioningArgs.builder().bucket("my-bucketname").build());
-
-      System.out.println("Bucket versioning is disabled successfully");
-
+      VersioningConfiguration config =
+          minioClient.getBucketVersioning(
+              GetBucketVersioningArgs.builder().bucket("my-bucketname").build());
+      if (config.status() == VersioningConfiguration.Status.OFF) {
+        System.out.println("Versioning on bucket 'my-bucketname' is not enabled");
+      } else if (config.status() == VersioningConfiguration.Status.ENABLED) {
+        System.out.println("Versioning on bucket 'my-bucketname' is enabled");
+      } else if (config.status() == VersioningConfiguration.Status.SUSPENDED) {
+        System.out.println("Versioning on bucket 'my-bucketname' is suspended");
+      }
```

Comment:
Can we use `switch` statement here?
Or maybe we can just use the `toString()` method and make this a single statement.

```suggestion
     String statusStr = config.status().toString();
     System.out.println("Versioning on bucket 'my-bucketname' is " + statusStr.isEmpty ? "Off" : statusStr);
```

Reply:
Simplified to just print the status.